tools/launchdev.sh

```diff
@@ -413,6 +415,10 @@ function st2start(){
             --config-file $ST2_CONF --register-all
     fi
 
+    if [ "$copy_examples" = true ]; then
+        st2 run packs.setup_virtualenv packs=tests
```

Comment:
I believe this will fail when auth is enabled in the dev config.  Can you test this scenario and make sure the launch script handle this gracefully and warn users that they have to setup the virtualenv on their own.